src/PrestaShopBundle/Resources/config/routing/admin/sell/customer_service/customer_threads.yml

```diff
@@ -88,3 +88,11 @@ admin_customer_threads_bulk_delete:
     _controller: PrestaShopBundle\Controller\Admin\Sell\CustomerService\CustomerThreadController::bulkDeleteAction
     _legacy_controller: AdminCustomerThreads
     #  _legacy_link: AdminCustomerThreads
+
+admin_customer_threads_contact_options_update:
+  path: /contact_options
+  methods: [ POST ]
+  defaults:
+    _controller: PrestaShopBundle\Controller\Admin\Sell\CustomerService\CustomerThreadController::contactOptionsUpdate
+    _legacy_controller: AdminCustomerThreads
+    #  _legacy_link: AdminCustomerThreads
```

Comment:
Why are you adding something commented ? Is this something you forgot to remove ?

Reply:
Its more like if we want to test it and make it override legacy controller we just need to uncomment this. But maybe its not needed for this page.